test/e2e/recovery_and_restart/recovery_and_restart_test.go

```diff
@@ -171,9 +171,15 @@ var _ = ginkgo.Describe("", func() {
 		}
 
 		ginkgo.By("Waiting for sleep pods to be running")
-		err = kClient.WaitForJobPodsRunning(dev, job1.Name, parallelism, 30*time.Second)
+		err = kClient.WaitForJobPodsRunning(dev, job1.Name, parallelism, 60*time.Second)
 		Ω(err).NotTo(gomega.HaveOccurred())
-		err = kClient.WaitForJobPodsRunning(dev, job2.Name, parallelism, 30*time.Second)
+		err = kClient.WaitForJobPodsRunning(dev, job2.Name, parallelism, 60*time.Second)
+		Ω(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Deleting sleep jobs")
+		err = kClient.DeleteJob(job1.Name, dev)
+		Ω(err).NotTo(gomega.HaveOccurred())
+		err = kClient.DeleteJob(job2.Name, dev)
 		Ω(err).NotTo(gomega.HaveOccurred())
 
 		ginkgo.By("Deleting sleep pods")
```

Comment:
I think this part can be removed. This is exactly why I filed YUNIKORN-1404. No need to individually delete pods (optionally, we can wait for them to disappear).

Reply:
I think we can resolve this with a later PR. For now, let's just get this merged.